src/test/java/com/shapesecurity/shift/parser/ParserTest.java

```diff
@@ -590,308 +582,321 @@ public void testLibrary() throws IOException, JsError {
-
-    // testFailure("1++", "Invalid left-hand side in assignment");
-    // testFailure("1--", "Invalid left-hand side in assignment");
-    // testFailure("++1", "Invalid left-hand side in assignment");
-    // testFailure("--1", "Invalid left-hand side in assignment");
-
-    testFailure("for((1 + 1) in list) process(x);", "Invalid left-hand side in for-in");
-    testFailure("[", "Unexpected end of input");
-    testFailure("[,", "Unexpected end of input");
-    testFailure("1 + {", "Unexpected end of input");
-    testFailure("1 + { t:t ", "Unexpected end of input");
-    testFailure("1 + { t:t,", "Unexpected end of input");
-    testFailure("var x = /\n/", "Invalid regular expression: missing /");
-    testFailure("var x = \"\n", "Unexpected token ILLEGAL");
-    testFailure("var if = 42", "Unexpected token if");
-    testFailure("i #= 42", "Unexpected token ILLEGAL");
-
-    // testFailure("i + 2 = 42", "Invalid left-hand side in assignment");
-    // testFailure("+i = 42", "Invalid left-hand side in assignment");
-
-    testFailure("1 + (", "Unexpected end of input");
-    testFailure("\n\n\n{", "Unexpected end of input");
-    testFailure("\n/* Some multiline\ncomment */\n)", "Unexpected token )");
-    testFailure("{ set 1 }", "Unexpected number");
-    testFailure("{ get 2 }", "Unexpected number");
-    testFailure("({ set: s(if) { } })", "Unexpected token if");
-    testFailure("({ set s(.) { } })", "Unexpected token .");
-    testFailure("({ set s() { } })", "Unexpected token )");
-    testFailure("({ set: s() { } })", "Unexpected token {");
-    testFailure("({ set: s(a, b) { } })", "Unexpected token {");
-    testFailure("({ get: g(d) { } })", "Unexpected token {");
-    testFailure("({ get i() { }, i: 42 })",
-        "Object literal may not have data and accessor property with the same name");
-    testFailure("({ i: 42, get i() { } })",
-        "Object literal may not have data and accessor property with the same name");
-    testFailure("({ set i(x) { }, i: 42 })",
-        "Object literal may not have data and accessor property with the same name");
-    testFailure("({ i: 42, set i(x) { } })",
-        "Object literal may not have data and accessor property with the same name");
-    testFailure("({ get i() { }, get i() { } })",
-        "Object literal may not have multiple get/set accessors with the same name");
-    testFailure("({ set i(x) { }, set i(x) { } })",
-        "Object literal may not have multiple get/set accessors with the same name");
+    testFailure("var x = /(s/g", 0, 0, 0, "Invalid regular expression");
+    testFailure("var x = /[a-z]/\\ux", 0, 0, 0, "Unexpected token ILLEGAL");
+    testFailure("var x = /[a-z\n]/\\ux", 0, 0, 0, "Invalid regular expression: missing /");
+    testFailure("var x = /[a-z]/\\\\ux", 0, 0, 0, "Unexpected token ILLEGAL");
+    testFailure("var x = /[P QR]/\\\\u0067", 0, 0, 0, "Unexpected token ILLEGAL");
+  }
+
+  @Test
+  @Ignore
+  public void testES6Failure() {
```

Comment:
Apparently. But our parser does not support it. Let me remind you that they are marked as ignored, meaning they are just a little better than a commented code with a TODO.